fluent-dom/src/overlay.js

```diff
@@ -71,8 +71,14 @@ export default function overlayElement(element, translation) {
     return;
   }
 
+  const whitelistedAttrs = element.hasAttribute('data-l10n-attrs') ?
```

Comment:
Perhaps call this `explicitlyAllowed`? Also, would `data-l10n-attributes` be better?